client/app/utils/i18n.js

```diff
@@ -13,35 +13,33 @@
 
 import { bind } from 'lodash';
 
-function isServer() {
-  return typeof window === 'undefined';
-}
-
-if (isServer()) {
-
-  // Initialize global.I18n
-  // In browser we initialize this in a script-tag manually
-  global.I18n = {}; // eslint-disable-line no-undef
-
-  // Load the translation bundle in the global.I18n variable.
-  // In browser the bundle is loaded in a separate script-tag.
-
-  try {
-    // The translation bundle will be loaded to the global I18n
-    // variable. Initialize the variable here.
-    require('../i18n/all.js'); // eslint-disable-line no-undef
-  } catch (e) {
-    console.warn("Can't load language bundle all.js"); // eslint-disable-line no-console
-  }
-}
+// Initialize global.I18n
+// This might be initialized already, because in browser
+// we do initialize this before the translations get loaded
+// in a separate script tag
+global.I18n = global.I18n || {}; // eslint-disable-line no-undef
```

Comment:
<img width="639" alt="screen shot 2016-05-16 at 11 40 18" src="https://cloud.githubusercontent.com/assets/53923/15284619/07d921cc-1b5b-11e6-9c28-5b419485d10c.png">

Are you sure? I get the above error.